run.go

```diff
@@ -177,8 +177,11 @@ func DefaultNamespaceOptions() NamespaceOptions {
 		{Name: string(specs.UserNamespace), Host: true},
 		{Name: string(specs.UTSNamespace), Host: true},
 	}
-	g := generate.New()
-	spec := g.Spec()
+	g, err := generate.New("linux")
```

Comment:
I'm fine with hard-coding this here, because any cross-platform buildah implementation would have to switch OSes at a higher level, and only call `DefaultNamespaceOptions` on Linux (since that function makes no sense on other OSes).

Reply:
Right, thats the conclusion I came to as well